### ISSUE TITLE
Suggestion for AZ-300T04_Lab_Mod01_Implementing Azure Logic Apps

### ISSUE THREAD

**Issue by tbkobberoe** [MicrosoftLearning/AZ-300-MicrosoftAzureArchitectTechnologies#74]
Could we change the Storage account created in Exercise 1 task 1 to a Storage (general purpose v2) instead of v2: As far as I know the v1 is "legacy" and when implementing storage today the recommendation is to use v2

**Comment by polichtm** [comment]
Would you mind clarifying why you consider v1 to be more appropriate in this particular case?
rgds
Marcin

**Comment by tbkobberoe** [comment]
No.,what I ask is to make it a v2 instead of v1 to be more up to date

**Comment by polichtm** [comment]
Apologies - typo on my part. Let's try it again:
https://docs.microsoft.com/en-us/azure/azure-monitor/platform/autoscale-common-scale-patterns

**Comment by tbkobberoe** [comment]
My reasons are that v1 is considered legacy, even in Azure Docs, and when I look at the 3 reasons for still using v1 copied from here [https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview](url)

- Your applications require the Azure classic deployment model. General-purpose v2 accounts and Blob storage accounts support only the Azure Resource Manager deployment model.

- Your applications are transaction-intensive or use significant geo-replication bandwidth, but do not require large capacity. In this case, general-purpose v1 may be the most economical choice.

- You use a version of the Storage Services REST API that is earlier than 2014-02-14 or a client library with a version lower than 4.x, and cannot upgrade your application

I don't see any reason to use v1 at this time. I know I'm nitpicking, but I think the labs and demos in any course should use the newest technologies. Hope you can see my point

**Comment by polichtm** [comment]
I tend to be cautious about upgrading a technology purely for the sake of upgrading - at least not without fully understanding the implications. In this case, the most relevant one (in my opinion) concerns pricing (in cases where the number of IOPS is high - as you pointed out in your response). While obviously these implications won't have any meaningful impact in the lab scenarios, I think there might be a benefit of illustrating these points to students.  
I'm certainly open to an update - but I'd also be interested in opinions of others that contribute to this repo
rgds
Marcin

**Comment by Williams-Nest** [comment]
I have run this course two times and each time I have in this exercise pointed out for my students to use v2 storage. In the material as well as on https://docs.microsoft.com/en-us/azure it is recommended to use v2 storage because v1 is legacy (and between the lines will not exist in the near future).

**Comment by outlookinsight** [comment]
I use situations like this to help the students think through the considerations of both approaches.  How to discover articles like the one above, showing reasons why you might choose v1 instead of v2, is a key skill from which many can benefit. Thanks, tbkobberoe, for linking the article.  In this particular case, there is no compelling requirement driving the need for v1, and v2 helps to future-proof the solution.  It's a great way to teach the students how to "fish" when faced with equally viable solution decisions that need to be made.  They will undoubtedly face them again!
